src/vs/workbench/contrib/extensions/browser/media/extensionsViewlet.css

```diff
@@ -111,7 +111,7 @@
 .extensions-viewlet.narrow > .extensions .extension-list-item > .icon-container > .icon {
 	width: 24px;
 	height: 24px;
-	padding-top: 8px;
+	padding-top: 12px;
```

Comment:
See comment above about removing this line once the container moves to flexbox 👍